content/gocourse-2019-05-melbourne.slide

```diff
@@ -668,6 +668,14 @@ Before adding any commits to `feature` locally run:
 - Bonus points: comprehensive tests
 
 
+* Test samples
```

Comment:
Might be worthwhile putting _before_ the labs so people can use these tests in the labs.

Reply:
Fair enough